content/applications/hr/recruitment/recruitment-flow.rst

```diff
@@ -0,0 +1,231 @@
+================
+Recruitment flow
+================
+
+When a prospective employee applies for a job in Odoo, there is a preconfigured process from the
+:ref:`initial inquiry <recruitment/new>` to the :ref:`creating of a new employee
+<recruitment/new-employee>` once hired. The following outlines the default recruitment process for
+Odoo's *Recruitment* application.
+
+.. important::
+   The following is based on Odoo's default recruitment pipeline. Be advised that if
+   :ref:`modifications are made <recruitment/customize-stages>` to the pipeline, the process
+   differs.
+
+.. _recruitment/new:
+
+New
+===
+
+At the start of the process, all applicants appear in the :guilabel:`New` stage on the
+:guilabel:`Applications` page, whether submitted online or if the applicant is :doc:`manually
+entered by a recruiter <add-new-applicants>`.
+
+When the applicant's card is created, Odoo automatically populates the
+:guilabel:`Subject/Application`, the :guilabel:`Applicant's Name`, :guilabel:`Email`, and
+:guilabel:`Mobile` number, on the applicant's card. This information is required when applying for
+a job position, by default.
+
+.. note::
+   If the website application form is modified, different fields may be populated, based on what
+   information is requested on the website.
+
+If the applicant entered any information in the *Short Introduction* section of the online
+application, it populates the :guilabel:`Application Summary` tab at the bottom of the applicant's
+card.
+
+Resumé
+------
+
+If a resumé was attached to the online application, it appears in the :guilabel:`Files` section of
+the chatter, and is also stored in the *Documents* application.
+
+To find the recruitment documents, navigate to the main :menuselection:`Documents` dashboard, and
+click the :guilabel:`Recruitment` folder on the left-hand side. All recruitment documents are stored
+within that folder.
+
+If the :ref:`CV Display <recruitment/cv-display>` option was enabled in the :ref:`Settings
+<recruitment/settings>` menu of the *Recruitment* app, the resumé appears on the applicant's card,
+on the right-hand side.
+
+.. note::
+   Depending on the browser zoom level, or size of the browser screen, the resumé may appear below
+   the main applicant card information.
```

Comment:
...and if it appears below, it appears as a PDF link/button -- it doesn't just readily present itself